mifosng-android/src/main/res/values/strings.xml

```diff
@@ -200,6 +200,7 @@
     <string name="add_payment_detail">Add Payment Detail</string>
     <string name="generate_new">Generate New</string>
     <string name="currency">Currency</string>
+    <string name="firststart_splash_screen">firststart</string>
```

Comment:
Add this string in `constans.java` file or add the `translatable=false`  attribute to it.